hazelcast/src/main/java/com/hazelcast/internal/networking/nonblocking/NonBlockingIOThreadingModel.java

```diff
@@ -51,8 +55,8 @@
 public class NonBlockingIOThreadingModel
         implements IOThreadingModel {
 
-    private final NonBlockingIOThread[] inputThreads;
-    private final NonBlockingIOThread[] outputThreads;
+    private volatile NonBlockingIOThread[] inputThreads;
```

Comment:
I would prefer the order `final`, `normal` and `volatile` for fields (to see faster which are immutable or if they need to be thread-safe).